elife/scripts/update-composer.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+log_file="/tmp/update-composer-$(date -u +'%Y-%m-%dT%H:%M:%SZ').log"
+
+set -o pipefail
+if ! composer self-update 2>&1 | tee "$log_file"; then
+    echo "Composer update failure detected."
+    if grep "Connection timed out" "$log_file"; then
+        echo "Retrying update..."
+        composer self-update
+    else
+        echo "Failure does not seem retriable."
```

Comment:
Is there any harm in just using the normal retry script though?

Reply:
The original command fails after 60 seconds, if there are similar failure we should check them out rather than adding time to the build. In an ideal world this will be pushed down into a base image rather than being performed continuously.

Reply:
Ok, so to be expanded if we see more problems.